Terasology/Cooking
# InGameHelp + Documentation

### Contains

* Added InGameHelp component to all items present in this module. Now information about them can be accessed through the help menu under "Cooking".
* Added a few classes to facilitate the previous item.
* Added documentation to all classes.

### How to Test

* Create a new world with the Cooking module enabled. Warning: Make sure to bump up the max amount of RAM allowed, or you may end up with a HeapMemory exception.
* Open up the help menu, and navigate over to the "Cooking" section.
* View the items by clicking on them, and see if they behave normally.
* When the previous tests are confirmed, proofread the documentation.

### Issues

None so far.
